hiring/remote_test/README.md

```diff
@@ -0,0 +1,18 @@
+# Remote test
+
+This is our current remote test for screening candidates. We provide a candidate with both a brief and a marking guide that clearly explains what we are looking for technically. The purpose of this test is to measure a candidates exposure to well known best practices and also in the absence of exposure measure their ability to find the answer.
```

Comment:
I think this needs the comma?

"well known best practices, and also in the absence"